wandb/sdk/launch/builder/build.py

```diff
@@ -447,6 +446,8 @@ def construct_gcp_registry_uri(
 
 
 def _parse_existing_requirements(launch_project: LaunchProject) -> str:
+    import pkg_resources
```

Comment:
Should this use `wini.utils` or `packaging`?

Reply:
oh this is some logic related to the launch feature, it's part of the code we package with the wandb python package. 
See here the launch docs to learn more about the feature: https://docs.wandb.ai/guides/launch